### ISSUE TITLE
Accessibility Problems With Examples 2.12

### ISSUE THREAD

**Issue by mgifford** [openlayers/ol2#374]
I'm just doing some basic testing of the examples provided.

Many of them have accessibility problems when run through - http://wave.webaim.org/toolbar

There are a number of closed accessibility issues that I assume have gotten into the core code, but not sure if the were  brought into the examples.

In openlayers/examples/animated_panning.html

I can navigate nicely between the +/- but all the images are missing alt text & the links are also the same (which can cause some confusion.

There's a good example here of a site with buttons & text that's using OpenLayers:
  http://drupal.org/node/1470282#comment-5798872

I couldn't access any of the Overlays on the right hand side.

The access keys here:
  openlayers-js/openlayers/examples/accessible.html

The accessible version should be able to have images, just properly presented.  Having a whole different "Accessible" version is the wrong approach.

I do like the improvements that have come in and am hoping that we can polish a few more of them off before there's a formal release.

There's code like - https://ircan-rican.gc.ca/projects/gcwwwmap-carte-geo/

Which has undergone some testing for accessibility that could be brought into this project.
